src/core/Makefile.am

```diff
@@ -31,13 +31,18 @@ include $(abs_top_nlbuild_autotools_dir)/automake/pre.am
 lib_LIBRARIES                       = \
     libopenthread-ftd.a               \
     libopenthread-mtd.a               \
+    libopenthread-radio.a             \
     $(NULL)
 
 CPPFLAGS_COMMON                     = \
     -I$(top_srcdir)/include           \
     $(OPENTHREAD_TARGET_DEFINES)      \
     $(NULL)
 
+libopenthread_radio_a_CPPFLAGS      = \
+    $(CPPFLAGS_COMMON)                \
+    -DOPENTHREAD_RADIO=1              \
```

Comment:
It will be only part of NCP app, right?
Some suggestions:
- `OPENTHREAD_RAW_LINK_NCP`
- `OPENTHREAD_RAW_LINK`